rlkit/core/rl_algorithm.py

```diff
@@ -411,11 +411,13 @@ def evaluate(self, epoch):
         ### sample trajectories from prior for debugging / visualization
         if self.dump_eval_paths:
             prior_paths = []
+            # (AZ): should this be hardcoded? also why is this # 100?
```

Comment:
same answer as below (debugging) - 100 because that's what looked pretty for point robot...

Reply:
got it, I'll add the comment